src/wmtk/attribute/AttributeManager.hpp

```diff
@@ -86,6 +86,9 @@ class AttributeManager : public wmtk::utils::MerkleTreeInteriorNode
 
     std::string get_name(const attribute::TypedAttributeHandleVariant& attr) const;
 
+    template <typename T>
+    void set_name(const TypedAttributeHandle<T>& attr, const std::string& name);
```

Comment:
just to be clear-  this should only be public in the manager class but the mesh class doesn't expose this right?

Reply:
Correct. It is not in the mesh class at all.